build.gradle

```diff
@@ -35,6 +37,7 @@ opensearchplugin {
     classname "${projectPath}.${pathToPlugin}.${pluginClassName}"
     licenseFile rootProject.file('LICENSE')
     noticeFile rootProject.file('NOTICE')
+    extendedPlugins = ['opensearch-job-scheduler']
```

Comment:
why this is required?

Reply:
You mean why we need `opensearch-job-scheduler`? or this `extendedPlugins` variable?
For the first, we use opensearch-job-scheduler to update datasource. For the second, it is needed to install opensearch-job-scheduler in OpenSearch cluster. See https://github.com/opensearch-project/neural-search/blob/main/build.gradle#L75